client/webserver/site/src/css/market_dark.scss

```diff
@@ -1,5 +1,5 @@
 body.dark {
-  input[type=number] {
+  input[type='number'] {
```

Comment:
Can you justify this change? From all appearances, the new syntax appears to be incorrect.

Reply:
it's my autosave prettier formatting, will revert.